packages/backend/src/QueryEngine.ts

```diff
@@ -13,17 +13,20 @@ import { SynthqlError } from './SynthqlError';
 export interface QueryEngineProps<DB> {
     url?: string;
     schema?: string;
+    prependSql?: string;
     providers?: Array<QueryProvider<DB, Table<DB>>>;
     pool?: Pool;
 }
 
 export class QueryEngine<DB> {
     private pool: Pool;
     private schema: string;
+    private prependSql: string;
```

Comment:
this shouldn't be global: it should be set on a per-request basis.

Reply:
Fair enough, will adjust now 👍🏾